magics.serq

```diff
@@ -0,0 +1,25 @@
+// Defines symbols that cannot be represented in pure serq code
+
+
+
+// Types
+
+@magic pub type int8
+@magic pub type uint8
+@magic pub type int16
+@magic pub type uint16
+@magic pub type int32
+@magic pub type uint32
+@magic pub type int64
+@magic pub type uint64
+
+@magic pub type float32
+@magic pub type float64
+
+@magic pub type bool
+
+@magic pub type string
+
```

Comment:
this is missing the char and array types

Reply:
They aren’t implemented, will be defined when they are